test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,18 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  it("returns string of names, seperated by commas and an ampersand", () => {
```

Comment:
This test is good, it might be easier to read if you assign your arguments to variables higher up. We also need to watch how things are laid out, usually when things are indented it means they are nested inside another function. We want all things at the same level of nesting at the same level of indentation.